crates/amaru-ledger/src/store/columns/dreps.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2025 PRAGMA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use amaru_kernel::{cbor, Anchor, Epoch, Lovelace, StakeCredential};
+use iter_borrow::IterBorrow;
+
+pub const EVENT_TARGET: &str = "amaru::ledger::store::dreps";
+
+/// Iterator used to browse rows from the Accounts column. Meant to be referenced using qualified imports.
+pub type Iter<'a, 'b> = IterBorrow<'a, 'b, Key, Option<Row>>;
+
+pub type Value = (Option<Anchor>, Option<Lovelace>);
```

Comment:
I don't think the DRep's deposit can ever be `None`? Unless I am missing something?

Reply:
That's indirectly due to `DiffBind#registered` type (`BTreeMap<K, (Option<J>, Option<V>)>`). I didn't want to update this structure as part of this PR but that would make sense to reconsider this once the global picture is known.